projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1404,3 +1412,37 @@ public static function register_block_metadata_collection() {
 	 */
 	add_filter( 'jetpack_block_editor_enable_upgrade_nudge', '__return_true' );
 }
+
+/**
+ * Overrides the `wp-block-editor` script with a hotfix to prevent a performance issue.
+ *
+ * @param WP_Scripts $scripts WP_Scripts instance.
+ */
+function jetpack_hotfix_block_editor_script( $scripts ) {
+	if ( ! defined( 'GUTENBERG_VERSION' ) ) {
+		return;
+	}
+
+	if ( ! in_array( GUTENBERG_VERSION, array( '19.9.0', '20.0.0', '20.1.0' ), true ) ) {
```

Comment:
Do we need to account for `GUTENBERG_DEVELOPMENT_MODE` as well?

Reply:
Yeah, good idea. Otherwise Gutenberg developers would have unexpected side effects if they also have Jetpack active. Done in https://github.com/Automattic/jetpack/pull/41309/commits/7da9eaa37701e2f8b05f495585a2ed0be52aaae7